agent/engine/docker_task_engine.go

```diff
@@ -508,6 +509,20 @@ func (engine *DockerTaskEngine) GetTaskByArn(arn string) (*api.Task, bool) {
 }
 
 func (engine *DockerTaskEngine) pullContainer(task *api.Task, container *api.Container) DockerContainerMetadata {
+	if container.Type == api.ContainerCNIPause {
+		// ContainerCNIPause images are managed at startup
+		return DockerContainerMetadata{}
```

Comment:
This is redundant.

Reply:
What part is redundant?  The comment?  That explains *why* there's an early return.

Reply:
The whole `if` block doesn't seem necessary. The `switch` block that runs the same check.

Reply:
Oh!  Yeah...oops.  Will remove!